pkg/linters/images/rules/images.go

```diff
@@ -358,3 +359,91 @@ type werfFile struct {
 	From     string `json:"from" yaml:"from"`
 	Final    *bool  `json:"final" yaml:"final"`
 }
+
+func checkImageNames(moduleName, path string) errors.LintRuleErrorsList {
+	result := errors.LintRuleErrorsList{}
+
+	imagesPath := filepath.Join(path, ImagesDir)
+
+	if !IsExistsOnFilesystem(imagesPath) {
+		return result
+	}
+
+	err := filepath.Walk(imagesPath, func(fullPath string, _ os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if fsutils.IsDir(fullPath) {
+			return nil
+		}
+
+		if filepath.Base(fullPath) == "werf.inc.yaml" {
+			lines, err := checkImageNamesFromFile(fullPath)
+			path, _ := filepath.Rel(imagesPath, fullPath)
+			if err != nil {
+				result.Add(errors.NewLintRuleError(
+					ID,
+					path,
+					moduleName,
+					nil,
+					"Cannot read file: %s",
+					err.Error(),
+				))
+				return nil
+			}
+			if len(lines) > 0 {
+				result.Add(errors.NewLintRuleError(
+					ID,
+					path,
+					moduleName,
+					nil,
+					"Image name format should be like `image: {{ .ModuleName }}/{{ .ImageName }}[-something]` in file: %s\n\t\t  broken lines:\n\t\t  %s",
+					path,
+					strings.Join(lines, "\n\t\t  "),
+				))
+			}
+		}
+
+		return nil
+	})
+	if err != nil {
+		result.Add(errors.NewLintRuleError(
+			ID,
+			imagesPath,
+			moduleName,
+			nil,
+			"Cannot read directory structure: %s",
+			err.Error(),
+		))
+
+		return result
+	}
+
+	return result
+}
+
+func checkImageNamesFromFile(path string) ([]string, error) {
+	var lines []string
+	file, err := os.Open(path)
+	if err != nil {
+		return lines, err
+	}
+	defer file.Close()
+
+	r := regexp.MustCompile(`^image:\s*\{\{\s*\$?\.ModuleName\s*\}\}/\{\{\s*\$?\.ImageName\s*\}\}`)
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "image:") {
+			if !r.MatchString(line) {
+				lines = append(lines, line)
+			}
+		}
+	}
```

Comment:
I guess we can add a line number to the output. It would be easier to find multiply selections:
```
🐒 [#images]
        Message - Image name format should be like `image: {{ .ModuleName }}/{{ .ImageName }}[-something]` in file: webui/werf.inc.yaml
                  broken lines:
                  image: {{ .ModuleName }}/build-webui-artifact (Line: 2)
                  image: {{ .ModuleName }}/foo (Line: 19)
        Object  - webui/werf.inc.yaml
        Module  - upmeter

```


```suggestion
        lineNumber := 0
	for scanner.Scan() {
	        lineNumber++
		line := scanner.Text()
		if strings.HasPrefix(line, "image:") {
			if !r.MatchString(line) {
				lines = append(lines, fmt.Sprintf("%s (Line: %d)", line, counter))
			}
		}
	}
```